client/web/src/repo/RevisionsPopover/RevisionsPopoverReferences.tsx

```diff
@@ -145,12 +144,22 @@ export const RevisionsPopoverReferences: React.FunctionComponent<
     const [searchValue, setSearchValue] = useState('')
     const query = useDebounce(searchValue, 200)
     const location = useLocation()
+    const { height } = useWindowSize()
+    const [batchCount, setBatchCount] = useState(getBatchCount(height))
+
+    /*
+     * Set batch count based on current screen height
+     * to avoid jumping tabs in the UI.
+     * */
+    useMemo(() => {
+        setBatchCount(getBatchCount(height))
+    }, [height])
```

Comment:
react n00b here: couldn't we do this more simply?

```suggestion
    /*
     * Set batch count based on current screen height
     * to avoid jumping tabs in the UI.
     * */
    const batchCount = useMemo(() => getBatchCount(height), [height]) 
```

Reply:
I'll try it out. Maybe @felixfbecker or @vovakulikov will know if this will work. My guess is that it won't but that's more instinct that actual knowledge.

Reply:
I think you're looking for @fkling

Reply:
Yup. This totally works.